app/policies/application_policy.rb

```diff
@@ -0,0 +1,54 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(:id => record.id).exists?
+  end
+
+  def create?
+    false
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    user.present?
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    false
+    # user.admin? || wiki.user?
```

Comment:
@cbeck I remembered this and tried to test it in latest checkpoint and it still looks like any user can delete an item